columnflow/plotting/plot_util.py

```diff
@@ -279,12 +282,12 @@ def prepare_plot_config(
             "kwargs": {
                 "norm": data_norm,
                 "label": "Data",
-                "yerr": False if data_hide_errors[i] else None,
-            },
-            "ratio_kwargs": {
-                "norm": h_mc.values() * data_norm / mc_norm,
-                "yerr": False if data_hide_errors[i] else None,
+                "yerr": False if data_hide_errors[0] else None,
             },
+            # "ratio_kwargs": {
+            #     "norm": h_mc.values() * data_norm / mc_norm,
```

Comment:
the "norm" parameter is quite important for the ratio plot, so we should included it back rather sooner than later. Otherwise, there will be no visible data points in the data/mc ratio. Suggestion, which adds it back as long as some stack is present.
```suggestion
            if h_mc:
                plot_config["ratio_kwargs"]["norm"] = h_mc.values() * data_norm / mc_norm
```